### PR TITLE
Ignoring auth tests to be fixed in GAWB-3215

### PR DESCRIPTION
**Pull request by andy7i**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala [hunk 1]
```diff
@@ -529,7 +529,8 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
       }
     }
 
-    "+ project owner, + group member, create workspace, - project owner" in withWebDriver { implicit driver =>
+    // Ignored as of 2/16/2018 - ticket GAWB-TBD
```
Comment (breilly2) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala]: Let's make the ticket and update this reference before disabling.

**Review by breilly2 — APPROVED** [review]
Reluctant 👍  assuming the ticket is made and there is a plan for manual testing until the automated tests are fixed.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala (hunk 1 above)
Comment (andy7i) [reply automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala]: Created and added

**Comment by MattPutnam** [comment]
👍 yes plz

**Pull request merged**